src/apps_plugin.c

```diff
@@ -245,6 +246,9 @@ long get_system_cpus(void) {
 
 long get_system_pid_max(void) {
 	procfile *ff = NULL;
+
+	// FIXME: Is this a typical maximum system pid count, or just for Linux?
+	//        Is it defined on POSIX.1 ABI X/Open and/or BSD specs?
 	long mpid = 32768;
```

Comment:
From 'man 5 proc':

"On 64-bit systems, pid_max can be set to any value up  to 2^22 
(PID_MAX_LIMIT, approximately 4 million)"

On 28-04-2016 13:48, Costa Tsaousis wrote:
>
> In src/apps_plugin.c 
> <https://github.com/firehol/netdata/pull/323#discussion_r61461979>:
>
> > @@ -245,6 +246,9 @@ long get_system_cpus(void) {
> >
> >  long get_system_pid_max(void) {
> >  	procfile *ff = NULL;
> > +
> > +	// FIXME: Is this a typical maximum system pid count, or just for Linux?
> > +	//        Is it defined on POSIX.1 ABI X/Open and/or BSD specs?
> >  	long mpid = 32768;
>
> man 5 proc
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/firehol/netdata/pull/323/files/460da7b0b8e59ebcb062b3b1627b452141c1a8ca..21b2e778bcce1d92816265f64175951556e53350#r61461979>
>